release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -144,6 +144,77 @@ Netlink messages dropped by OVS kernel module due to netlink socket buffer overf
+
+While it is acceptable to have multiple default storage classes for a short time, you should ensure that eventually only one default storage class exists.
+
+For more information, see ???.
+
+==== Managing the default storage class is generally available 
+{product-title} 4.13 introduces the `spec.storageClassState` field in the ClusterCSIDriver object, which allows you to manage the default storage class to accomplish several different objectives:
+
+* Enforcing static provisioning by disabling dynamic provisioning.
+
+* When you have other preferred storage classes, preventing the storage operator from re-creating the initial default storage class.
+
+* Renaming, or otherwise changing, the default storage class
+
+For more information, see ???.
+
+==== Retroactive default StorageClass assignment (Technology Preview)
+There are some cases where the default storage classes are not assigned, or more than one default is defined. This feature allows you to retroactively assign a default storage class to correct theses situations. This feature is supported with Technology Preview status.
+
+For more information, see ???.
+
+==== CSI inline ephemeral volumes is generally available
+Container Storage Interface (CSI) inline ephemeral volumes were introduced in {product-title} 4.5 as a Technology Preview feature, and allows you to define a Pod spec that creates inline ephemeral volumes when a pod is deployed and delete them when a pod is destroyed. This feature is now generally available.
+
+This feature is only available with supported Container Storage Interface (CSI) drivers.
+
+For more information, see xref:../storage/container_storage_interface/ephemeral-storage-csi-inline.adoc[CSI inline ephemeral volumes].
+
+==== Automatic CSI migration for Microsoft Azure File auto migration is generally available
+Starting with {product-title} 4.8, automatic migration for in-tree volume plugins to their equivalent Container Storage Interface (CSI) drivers became available as a Technology Preview feature. Support for Azure File was provided in this feature in {product-title} 4.10, and {product-title} 4.13 now supports automatic migration for Azure File as generally available. CSI migration for Azure File is now enabled by default and requires no action by an administrator.
+
+This feature automatically translates in-tree objects to their counterpart CSI representations and should be completely transparent to users. Translated objects are not stored on disk, and user data is not migrated.
+
+While storage class referencing to the in-tree storage plugin will continue working, it is recommended that you switch the default storage class to the CSI storage class.
```

Comment:
ISG says only use `while`to _refer to a period of time_.  Suggested `although` or `though`

`storage class referencing to the in-tree storage plugin` 
This is probably right. My ignorance of the subject makes it sound odd. 

s/will continue/continues